server/api/templates/docs/pd.swagger.json

```diff
@@ -0,0 +1,809 @@
+                        "description": "A regions object.",
+                        "schema": {
+                            "$ref": "#/definitions/Regions"
+                        }
+                    },
+                    "500": {
+                        "description": "unexpected error"
+                    }
+                }
+            }
+        },
+        "/pd/api/v1/region/{id}": {
+            "get": {
+                "description": "Get a TiKV region.",
+                "parameters": [{
+                    "name": "id",
+                    "type": "integer",
+                    "in": "path",
+                    "required": true,
+                    "description": "The id of the region to get."
+                }],
+                "responses": {
+                    "200": {
+                        "description": "A region object.",
+                        "schema": {
+                            "$ref": "#/definitions/Region"
+                        }
+                    },
+                    "500": {
+                        "description": "unexpected error"
+                    }
+                }
+            }
+        },
+        "/pd/api/v1/events": {
+            "get": {
+                "description": "Get all PD events.",
+                "responses": {
+                    "200": {
+                        "description": "An array of event objects.",
+                        "schema": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/definitions/LogEvent"
+                            }
+                        }
+                    },
+                    "500": {
+                        "description": "unexpected error"
+                    }
+                }
+            }
+        },
+        "/pd/api/v1/balancers": {
+            "get": {
+                "description": "Get all PD balancers.",
+                "responses": {
+                    "200": {
+                        "description": "A balancers object.",
+                        "schema": {
+                            "$ref": "#/definitions/Balancers"
+                        }
+                    },
+                    "500": {
+                        "description": "unexpected error"
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "Version": {
+            "type": "object",
+            "properties": {
+                "version": {
+                    "type": "string",
+                }
+            }
+        },
+        "Balance": {
+            "type": "object",
+            "properties": {
+                "min-capacity-used-ratio": {
+                    "type": "number",
+                    "format": "double",
+                    "description": "For capacity balance. If the used ratio of one store is less than this value, it will never be used as a from store."
```

Comment:
For capacity balance. If the used ratio of one store is less than this value, it will never be used as a source store (from_store).